doc/Messages.md

```diff
@@ -127,6 +127,21 @@ Passes the user's option values from content to background for persistence.  Req
 
 * `excludes` A string, one `@exclude` pattern per line.
 
+# UserScriptGetByName
```

Comment:
We call this (namespace+name) the "ID".

I think we should keep the existing UserScriptGet message, and let it accept either an ID or a UUID.